src/xrCore/cpuid.cpp

```diff
@@ -163,7 +163,7 @@ unsigned int query_processor_info(processor_info* pinfo)
     unsigned int pa_mask_save = 0;
     cpu_set_t my_set;
     CPU_ZERO(&my_set);
-    sched_getaffinity(0, sizeof(cpu_set_t), &my_set);
+    pthread_getaffinity_np(pthread_self(), sizeof(cpu_set_t), &my_set);
```

Comment:
And rehe you do the opposite: `sched_` function is replaced with `pthread_`

Reply:
sched_getaffinity is a Linux-ism whereas pthread_getaffinity_np exists outside of Linux's libc.

Reply:
Hm.. This is weird. But let it be.

Reply:
sched_getaffinity is Linux-specific as per its man page
pthread_getaffinity_np (np is for non-portable) is GNU specific as per its man page